lib/cpp/test/qt/CMakeLists.txt

```diff
@@ -18,11 +18,15 @@
 #
 
 set(CMAKE_AUTOMOC ON)
-find_package(Qt5 REQUIRED COMPONENTS Test)
+find_package(Qt5 REQUIRED COMPONENTS Test Network)
```

Comment:
Network module should be already found either by non-test part of cpp/CMakeLists.txt or qt5/CMakeLists.txt before reaching here.
I'm not particularly against being explicit if it's your intention, though.